### DISCUSSION TITLE
Cup & Tour Scoring Estimates

### DISCUSSION THREAD

**Discussion by MktDad** [MKT-Playbook/Hub#39] [Feature requests]
I would love it if each cup and the whole tour had scoring estimates that worked the same way that single races worked. Let me enter a % of scoring that I think I can achieve (I use 85%) and then tell me what cumulative score I can expect in each cup and most importantly for the tour. I’d like to compete for Top 1000 but I don’t know if I can achieve those sorts of scores….not without doing separate math (and who wants to do that?).

**Comment by TitusPosiadly** [comment]
Hey, thanks for the suggestion.
Yes! I think the ACR tool is the next big thing on my list. First, the new version will have to be released though (not including the ACR Tool, but a better coverage tool). I'm in the last phase of Beta testing and will hopefully release version 1.3 in one or two weeks. Once that is all done I can start working on the ACR tool.

**Reply by StoneyX-1** [reply]
Hi,  I believe my  QOL Suggestion is similar enough to this one.  Not for ACR but for each cup (ranked one specifically).

Just a summary of the estimated scores for the 3 so it’s easier and quicker to see how your current score in game is vs total on the ranked board.  

The ACR as well, but I assume that would require more of a UI change than adding for the individual cups

**Reply by TitusPosiadly** [reply]
Hey, yes! That is a suggestion I already have high on my to-do list, but even the 'simple' summary of all 3 score is unfortunately not as easy to implement as I would like. At least not with how the app is set up at the moment. On top of that, I currently have very limited time, if any at all, to work on the app. But like I said, it is on my to-do list and I will try to work on the app some more when I get the time. For right now, I pushed it up even further on the priority, so thank you very much for suggesting it.
